tests/test_buffer.py

```diff
@@ -8,6 +8,21 @@
 
 class TestBufferHistory(TestCase):
     def test_buffer_history(self):
+        """
+        This test is to check the buffer asset and its interaction with the network.
+
+        Checks:
+        - The logic of how the history is set for the buffer. This is to avoid artificial energy
+        at t=0.
+        - Checks that when this is overwritten this artificial energy is indeed exploited.
```

Comment:
When what is overwritten? "- Checks that when this is overwritten this artificial energy is indeed exploited."

Reply:
let's leave this for the moment, this test will be redone anyway